samples/web/content/apprtc/js/main.js

```diff
@@ -358,3 +399,142 @@ function toggleFullScreen() {
     trace(event);
   }
 }
+
+roomIdInput.addEventListener('input', function(){
```

Comment:
Could you move the room related code into a separate file and make it more object oriented? E.g. what https://github.com/GoogleChrome/webrtc/pull/316 tries to do.
Also please avoid using global data variables.